independent-projects/arc/processor/banned-signatures.txt

```diff
@@ -0,0 +1,8 @@
+@defaultMessage Don't use Maven classes. They won't be available when using Gradle.
+org.apache.maven.**
+org.codehaus.plexus.**
+
+@defaultMessage Never use Type#toString() as it's almost always the wrong thing to do. Usually org.jboss.jandex.DotName#toString() is what is needed
+org.jboss.jandex.Type#toString()
```

Comment:
why having this 3 times now ?

<img width="1370" alt="image" src="https://github.com/user-attachments/assets/6a75e33d-06a8-403e-9073-524f07f09f26" />

<img width="1382" alt="image" src="https://github.com/user-attachments/assets/3d853c01-8122-4019-87fe-6047b7b368ac" />

Reply:
config can be merged into `banned-signatures-common.txt`:
<img width="1685" alt="image" src="https://github.com/user-attachments/assets/3534277e-c4d8-4ade-a921-d1e095138c8a" />